homeassistant/components/imap_email_content/sensor.py

```diff
@@ -231,26 +232,30 @@ def get_msg_text(email_message):
         for part in email_message.walk():
             if part.get_content_type() == CONTENT_TYPE_TEXT_PLAIN:
                 if message_text is None:
-                    message_text = part.get_payload()
+                    message_text = part.get_payload(decode=True)
```

Comment:
Maybe the ```decode``` value could be read from config.
With a default ```decode``` to False, users who don't need the payload to be decoded or decode it elsewhere not impacted by this update